app/controllers/api/v1/licenses/relationships/owners_controller.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+module Api::V1::Licenses::Relationships
+  class OwnersController < Api::V1::BaseController
+    before_action :scope_to_current_account!
+    before_action :require_active_subscription!
+    before_action :authenticate_with_token!
+    before_action :set_license
+
+    authorize :license
+
+    def show
+      owner = license.owner
+      authorize! owner,
+        with: Licenses::OwnerPolicy
+
+      render jsonapi: owner
+    end
+
+    typed_params {
+      format :jsonapi
+
+      param :data, type: :hash, allow_nil: true do
+        param :type, type: :string, inclusion: { in: %w[user users] }
+        param :id, type: :uuid
+      end
+    }
+    def update
+      owner = license.owner
+      authorize! owner,
+        with: Licenses::OwnerPolicy
+
+      license.update!(user_id: owner_params[:id])
+
+      BroadcastEventService.call(
+        event: 'license.owner.updated',
+        account: current_account,
+        resource: license,
+      )
+
+      # FIXME(ezekg) This should be the user
```

Comment:
Technically, this should be the relationship linkage, but since this isn't under the `/relationships` namespace, it's a gray area. I still think that returning the license object here (and in other relationships) feels weird. Maybe we should update this to be the `owner` for both here and the machine's owner controller? Our users controller already returns the attached users for `#attach` so might as well take advantage of the fact we can fix our wrong here i.r.t. deprecating the old user controllers and replacing with these owner controllers.

Reply:
If we do go this route, changing the response, we'll need to assert the bearer has `user.read` permissions (which would be assumed since they also have `license.owner.update` permissions.

What would be nice here would be a `render jsonapi: ..., linkage: true` option, where only linkage is rendered.

Reply:
I'm going to disregard this for now. It's too much to diverge from what our other relationship endpoints return. Even if it's not perfect, it's what we have. If it's that much of a problem, we can always write a migration to fix it later.